notebooks/COS/Requirements.txt

```diff
@@ -0,0 +1,20 @@
+## Requirements
```

Comment:
```suggestion
```
This isn't harmful since it's a comment character, but it  isn't really necessary since the file is already named "requirements"